examples/doc.txt

```diff
@@ -0,0 +1,7 @@
+/**
+@defgroup examples Mesh4all Examples
+
+## This is example-doc
```

Comment:
this should be developed in the near future :+1:

Reply:
Do you refers to last file changed?